js/lib/MerkleTree.js

```diff
@@ -53,6 +53,10 @@ class MerkleTree {
     return bufferToHex(this.getRoot());
   }
 
+  getProofByElem (el) {
```

Comment:
we follow the [airbnb styleguide](https://github.com/airbnb/javascript). perhaps it would be good to add es lint or prettier to the project. however until then there shouldn't be a space between the name and the `()`

Reply:
@lchangdev could you add the linter that we use for other projects in a separate PR? Shouldn't hurt anything.